keras/backend/mxnet_backend.py

```diff
@@ -3037,7 +3037,15 @@ def conv3d(x, kernel, strides=(1, 1, 1), padding='valid',
     # Raises
         ValueError: if `data_format` is neither `channels_last` or `channels_first`.
     """
-    raise NotImplementedError('MXNet Backend: conv3d operator is not supported yet.')
+    if data_format is None:
+        data_format = image_data_format()
+    _validate_data_format(data_format)
+
+    if padding not in {"same", "valid"}:
+        raise ValueError("`padding` should be either `same` or `valid`.")
```

Comment:
Please use ' instead of " for all strings.

Reply:
we can use [pep8](https://pypi.python.org/pypi/pep8) to check coding style

Reply:
@sandeep-krishnamurthy Done. Thank-You.

@roywei yes absolutely